tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -90,6 +91,34 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("Plugin should translate child plan of write commands to GPU") {
+
+    val conf = new SparkConf()
+      .set(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "true")
+      .set(SQLConf.ADAPTIVE_EXECUTION_FORCE_APPLY.key, "true")
+      // DataWritingCommandExec does not get translated to GPU because the plugin
+      // currently doesn't support the CreateDataSourceTableAsSelectCommand command
```

Comment:
This comment should be updated to explain why we're explicitly disabling the exec, maybe calling out that Spark AQE doesn't yet work with `GpuDataWritingCommandExec` and linking to the issue.